app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1870,29 +1847,14 @@ class BrowserTabFragment :
             newTabLayout.setOnClickListener(null)
         }
 
-        private fun showHomeTopCta(configuration: HomeTopPanelCta) {
```

Comment:
If we are removing the home top panel functionality, does it make sense to also remove all the references to it?
- `HomeTopPanelCta` sealed class
- References in the `BrowserTabViewModel`
- `include_top_cta.xml` layout

Reply:
I was in doubt about this one. While I was ok removing parts that were slowing down the startup process + som warnings caused by that. I was ok moving the final cleanup into a LHF task. But since you asked for that, and you know about that HomeTopCta, I will remove it 100%

Reply:
Yeah, we don't use it anymore and I doubt we ever will 👍